linera-core/src/client/mod.rs

```diff
@@ -1259,8 +1267,13 @@ where
                     .await?
                 {
                     HandleCertificateResult::FutureEpoch => {
-                        warn!("Postponing received certificate from {:.8} at height {} from future epoch {}",
-                              chain_id, certificate.value().height(), certificate.value().epoch());
+                        warn!(
+                            "Postponing received certificate from {:.8} at height {} \
+                            from future epoch {}",
+                            chain_id,
+                            certificate.value().height(),
+                            certificate.value().epoch()
+                        );
```

Comment:
I wonder why it didn't format it for me w/ `cargo fmt` (or that the CI doesn't complain about it).

Reply:
I think `rustfmt` gives up if a string literal is too long.